### DISCUSSION TITLE
DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ek1ng/blog#26] [Announcements]
# DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog

DAS的练手小比赛啦，打的不太好，继续沉淀 - ek1ng - ek1ng's Blog

https://ek1ng.com/DASCTF2022_7.html

**Comment by local-h0st** [comment]
昨天刚打完，我也是web选手，高强度坐牢看了一天题也只做了签到。倒是队内有大哥又kill了两道web，只能说是我自己水平还比较菜。今天搜writeup搜到你这个博客了，先好好学习一下，然后尝试复现

**Comment by local-h0st** [comment]
不对啊你blog没有其他题的writeup吗😂，抱歉打扰了

**Reply by ek1ng** [reply]
当时只在下午看了一会题目，其他题目复现了但是博客还没更新哈哈哈，比赛时没做出来，可以加个联系方式交流一下哇

**Comment by local-h0st** [comment]
嗯，共勉！



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年7月25日(周一) 下午4:34
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: [ek1ng/blog] DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog (Discussion #26)




 
DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog
 
DAS的练手小比赛啦，打的不太好，继续沉淀 - ek1ng - ek1ng's Blog
 
https://ek1ng.com/DASCTF2022_7.html

**Comment by local-h0st** [comment]
过几天接着来看你博客writeup



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年7月25日(周一) 下午4:34
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: [ek1ng/blog] DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog (Discussion #26)




 
DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog
 
DAS的练手小比赛啦，打的不太好，继续沉淀 - ek1ng - ek1ng's Blog
 
https://ek1ng.com/DASCTF2022_7.html

**Comment by sanyue-github** [comment]
确实，题目最主要的还是环境真实。然后newser我都不知道怎么复现，他怎么找出来那两个文件的，太强了

**Comment by local-h0st** [comment]
已收到，谢谢师傅的writeup，共勉



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年8月13日(周六) 下午4:05
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [ek1ng/blog] DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog (Discussion #26)




 
确实，题目最主要的还是环境真实。然后newser我都不知道怎么复现，他怎么找出来那两个文件的，太强了

**Comment by local-h0st** [comment]
现在web都卷的要死，会pwn的才是真爷爷，哭



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年8月13日(周六) 下午4:05
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [ek1ng/blog] DASCTF 2022 7月赋能赛 writeup - ek1ng's Blog (Discussion #26)




 
确实，题目最主要的还是环境真实。然后newser我都不知道怎么复现，他怎么找出来那两个文件的，太强了
